ydb/public/api/protos/draft/persqueue_error_codes.proto

```diff
@@ -50,5 +50,7 @@ enum EErrorCode {
 
     UNKNOWN_TXID = 30;
 
+    PRECONDITION_FAILED = 31;
```

Comment:
New code. In Write session this code is converted to YdbStatus. You need to update this code too.

Reply:
it is in the file `persqueue_utils.cpp`